tests/app/data/docker/manifests-delta/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2.1'
```

Comment:
Once we are requiring 2.3 in [other parts of CI](https://github.com/mendersoftware/integration/pull/2314), I think is okey to modernize here too :+1: